src/main/java/com/percero/agents/auth/services/AnonAuthProvider.java

```diff
@@ -0,0 +1,56 @@
+package com.percero.agents.auth.services;
```

Comment:
Let's use an ActiveStack package for all new classes.  Something like:
org.activestack.auth.provider